content/installation/administrator/admin_install_r/Rprofile.site

```diff
@@ -7,40 +11,24 @@ options(
   xpinch = 300,
   ypinch = 300,
   yaml.eval.expr = TRUE,
-  usethis.full_name = "Research Institute for Nature and Forest",
-  usethis.description = list(
-    `Authors@R` = "c(
-  person(
-    \"Voornaam\", \"Achternaam\", role = c(\"aut\", \"cre\"),
-    email = \"voornaam.achternaam@inbo.be\",
-    comment = c(ORCID = \"9999-9999-9999-9999\")),
-  person(
-    \"Research Institute for Nature and Forest\",
-    role = c(\"cph\", \"fnd\"), email = \"info@inbo.be\"))",
-    License = "GPL-3",
-    Language = "en-GB",
-    Encoding = "UTF-8",
-    Roxygen = "list(markdown = TRUE)"
-  ),
+  lintr.linter_file = system.file("lintr", package = "checklist"),
   repos = c(
     CRAN = "https://cloud.r-project.org/",
-    INLA = "https://inla.r-inla-download.org/R/testing",
-    inbo = "https://inbo.r-universe.dev",
-    ropensci = 'https://ropensci.r-universe.dev',
-    thierryo = "https://thierryo.r-universe.dev"
+    INLA = "https://inla.r-inla-download.org/R/stable",
+    inbo = "https://inbo.r-universe.dev"
```

Comment:
Maybe keep the ropensci universe?

Reply:
`ropensci` can contain beta versions of packages. E.g. versions that are in the main branch on github but not yet publish on CRAN. Do we want everyone to install such versions by default?

Reply:
I honestly dislike it when R asks to update to develop versions while installing a package (and I hate it if I'm forced to do so), so I'm certainly not in favour of that.

Reply:
OK to keep it as is. Better not include ropensci in that case.